### ISSUE TITLE
Since the latest update, impossible to launch the debuger

### ISSUE THREAD

**Issue by Crash--** [Microsoft/vscode-react-native#309]
## Actual Behavior
1. First I had to reconfigure the environnements after the upgrade 
2. When I want to run the debuger I got "Le processus de l'adapptateur de débogage s'est terminé de manière inattendue" "Ouvrir launch.json" 

## Expected Behavior
1. I want the debuger to be launched!

## Software versions
- React-Native VS Code extension version: RN 0.33 / React Native Tools 0.2.0
- VSCode version: Version 1.5.3 (1.5.3)
- OS platform and version: macOS 10.11.6
- React Native version: RN 0.33 

## Outputs (Include if relevant)
- Output of the Debug Console (View -> Toggle Debug Console): 
- Output of the React-Native output channel (View -> Toggle Output -> Select React-Native in ListBox): 
- Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab): 

```
messageService.ts:119Le processus de l'adaptateur de débogage s'est terminé de manière inattendue : Error: Le processus de l'adaptateur de débogage s'est terminé de manière inattendue
    at Object._ [as create] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:34:29245)
    at file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:119:8772
    at Object.m [as _notify] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:13448)
    at Object.B.enter (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:16953)
    at n.Class.define._run (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:18274)
    at n.Class.define._chainedError (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:17633)
    at n (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:15456)
    at n.Class.define.then (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:19083)
    at Object.R.enter (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:15525)
    at n.Class.define._run (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:18274)
    at n.Class.define._setCompleteValue (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:30:17968)e.doShow @ messageService.ts:119
messageService.ts:119Le processus de l'adaptateur de débogage s'est terminé de manière inattenduee.doShow @ messageService.ts:119
```

**Comment by girvo** [comment]
It might be unrelated, but I ran into a similar bug. Disabling the "Flow" language support extension (or `vscode-flow`, depending) made it work again for me. Might not be the same bug, but thought I'd mention it just in case!

**Comment by Crash--** [comment]
Nice catch. I just removed an extension with Flow support and now it works! 

Thanks @girvo

**Comment by MSLaguana** [comment]
What version of vscode are you running? This issue was supposed to be fixed in VSCode 1.5

If you are using the latest version of VSCode, could you please report the issue at https://github.com/Microsoft/vscode/issues/10148

**Comment by Crash--** [comment]
I'm using the Version 1.5.3 (1.5.3). 

I'll report to #10148
